runners/tez/src/main/java/org/apache/beam/runners/tez/translation/ViewCreatePCollectionViewTranslator.java

```diff
@@ -0,0 +1,18 @@
+package org.apache.beam.runners.tez.translation;
+
+import org.apache.beam.sdk.transforms.View;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * {@link View.CreatePCollectionView} translation to Tez equivalent.
+ */
+class ViewCreatePCollectionViewTranslator<ElemT, ViewT> implements
+    TransformTranslator<View.CreatePCollectionView<ElemT, ViewT>> {
+  private static final Logger LOG = LoggerFactory.getLogger(ViewCreatePCollectionViewTranslator.class);
+
+  @Override
+  public void translate(View.CreatePCollectionView<ElemT, ViewT> transform, TranslationContext context) {
+    //TODO: Translate transform to Tez and add to TranslationContext
```

Comment:
<!--new_thread; commit:346274d62540a16c9bf795fc6e9ab2cd28430008; resolved:0-->
Should throw instead of being a noop. This can be captured by the overarching "find the TODOs" JIRA or you could fix it quickly.